collections/infrastructure/roles/pxe_stack/README.md

```diff
@@ -147,6 +147,7 @@ for a group of hosts or even at hostvars level.
 
 ## Changelog
 
+* 1.8.1: Update to BB 2.0 format again
```

Comment:
Please add your name to this change, for reference 🙂 Same for the other roles.

Reply:
Done too ;)